### DISCUSSION TITLE
Nuxt 3 + tRPC!

### DISCUSSION THREAD

**Discussion by wobsoriano** [trpc/trpc#1905] [Show and tell]
Published a Nuxt 3 module that lets you build  & consume fully typesafe APIs! Also uses Nuxt's [`useAsyncData`](https://v3.nuxtjs.org/api/composables/use-async-data) to take advantage of its features.

https://github.com/wobsoriano/trpc-nuxt

Feedbacks are welcome 😃

**Comment by sachinraja** [comment]
Amazing to see tRPC being used in more and more places! We'd love a PR to add this to https://trpc.io/docs/awesome-trpc
